modules/core/csv/src/FarmCsvServiceProvider.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Drupal\farm_csv;
+
+use Drupal\Core\DependencyInjection\ContainerBuilder;
+use Drupal\Core\DependencyInjection\ServiceModifierInterface;
+
+/**
+ * Overrides the CSV Serialization service class.
+ */
+class FarmCsvServiceProvider implements ServiceModifierInterface {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function alter(ContainerBuilder $container) {
```

Comment:
Would it make sense for this to decorate the existing implementation rather than be a straight replacement so other customizations could also be applied without being mutually exclusive with this one?

Reply:
Resolving this conversation because a different approach is being taken, which no longer replaces this class.